test/basic.test.js

```diff
@@ -110,6 +110,7 @@ test('cmd - test collect - system info, data files and html', function (t) {
 })
 
 test('cmd - test collect - does not crash on webassembly frames', function (t) {
+  t.jobs = 2
```

Comment:
I don't think this does what you think it does. There are no substests to run in parallel here.

Reply:
ok thanks. maybe I was just lucky that it passed 4 times in a row.  I thought it was allowing a required parallel process to run within the specific test.

Reply:
Well, let's land this and just see. This one has been troublesome for a while